firstvoices/backend/serializers/fields.py

```diff
@@ -88,3 +88,10 @@ def to_representation(self, value):
             return visibility_map[value]
         except KeyError:
             raise serializers.ValidationError("Invalid visibility value.")
+
+
+class NonNullCharField(serializers.CharField):
```

Comment:
This is a great solution and implementation. Nice!

My only quibble is about the name. This is a CharField that allows null input, so maybe NullableCharField makes more sense?

Reply:
No need to change it now, and no need to re-review if you do!